substratools/function.py

```diff
@@ -71,23 +72,12 @@ def _parser_add_default_arguments(parser):
     )
 
 
-class GenericAlgo(abc.ABC):
-    chainkeys_path = None
```

Comment:
why delete this?

Reply:
I mean `chainkeys_path = None`, did you check that `self.chainkeys_path` exists in the algo? 
i saw discussions on the backend to completely remove it, maybe we can remove it here too, but if not we need to check this variable is defined

Reply:
In which algo ? There is a field chainkeys path set is the FunctionWorkspace (previously AlgoWorkspace): 
`self.chainkeys_path = chainkeys_path or self._get_default_path(DEFAULT_CHAINKEYS_PATH)`. Is it what you are refering to?